tapir/shifts/views/calendars.py

```diff
@@ -136,12 +141,28 @@ def get_context_data(self, **kwargs):
 class ShiftTemplateGroupCalendar(LoginRequiredMixin, TemplateView):
     template_name = "shifts/shift_template_group_calendar.html"
 
-    def get_context_data(self, **kwargs):
-        context = super().get_context_data()
-        date_to_group = {}
-        today = timezone.now().date()
-        for week in range(52):
-            monday = today - timedelta(days=today.weekday()) + timedelta(weeks=week)
-            date_to_group[monday] = get_week_group(monday).name
-        context["date_to_group"] = date_to_group
+    def get_context_data(self, *args, **kwargs):
+        context = super().get_context_data(*args, **kwargs)
+
+        # TODO Frederik: a static calendar.html would accelerate
```

Comment:
How long does this take to render? If it's an issue, we should use the Django per-view cache, see https://docs.djangoproject.com/en/4.0/topics/cache/#the-per-view-cache

Reply:
We might have to change `get_week_group` as Theo said, but maybe see first how it performs